test/chpldoc/links/nested.doc.chpldocopts

```diff
@@ -0,0 +1 @@
+-o docs --no-html --save-sphinx docs
```

Comment:
Why use `--no-html` if you're going to call `sphinx-build` with the html flag later?

Reply:
I was getting errors from sphinx when not using `--no-html`, which I think are related to using `--save-sphinx` to the same directory as `-o`. Since for this test the sphinx output is not needed, I can simplify this (and the prediff)